lib/hexpm/accounts/audit_log.ex

```diff
@@ -138,6 +138,15 @@ defmodule Hexpm.Accounts.AuditLog do
   defp extract_params("password.reset.finish", nil), do: %{}
   defp extract_params("password.update", nil), do: %{}
 
+  defp extract_params("billing.create", params),
+    do: %{
+      email: params["email"],
```

Comment:
Do we need to convert these keys from strings to atoms like this? Or we can just leave them as strings? 
If we need to, is there any better ways to convert the keys?

Reply:
You can do it like this:

```
defp extract_params("billing.create", {organization, params}) do
  %{
    organization: organization,
    person: params["person"],
    company: params["company"],
    token: params["token"],
    quantity: params["quantity"]
  }
end
```

Reply:
updated as the snippet above, please check

Reply:
Looks great.